div4.py

```diff
@@ -0,0 +1,54 @@
+
+def isdiv4(number):
+
+#    number = "0" + number
+#    print(number)
+    tendigit = number[-2]
```

Comment:
tendeigit must be tensdigit OR separate words with underscore like tens_digit . Implement this for all cases.